FlowCrypt/Controllers/ImportKey/Enter Pass Phrase/EnterPassPhraseViewController.swift

```diff
@@ -251,6 +251,7 @@ extension EnterPassPhraseViewController {
     }
 
     private func moveToMainFlow() {
+        // TODO: - ANTON - enter pass phrase
```

Comment:
Is this still to be done?

Reply:
This all are just markers for me to no to forget to resolve before PR will be ready